libs/src/services/creatorNode/CreatorNodeSelection.test.ts

```diff
@@ -807,7 +807,7 @@ describe('test CreatorNodeSelection', () => {
     }
     // Make sure there is some variance
     assert(!primaries.every((val) => val === primaries[0]))
-  }).timeout(10000)
+  }, 10000)
```

Comment:
👀 ?

Reply:
That's the "preferred" way to define timeouts for tests

Reply:
🧠